File: apps/auth-server/src/main/java/com/example/chat/auth/server/common/exception/AuthException.java
package com.example.chat.auth.server.common.exception;

import com.example.chat.common.core.exception.BaseException;
import com.example.chat.common.core.exception.ErrorCode;

public class AuthException extends BaseException {
    public AuthException(ErrorCode errorCode) {
        super(errorCode);
    }

    public AuthException(ErrorCode errorCode, Object[] args) {
        super(errorCode, args);
    }

    public AuthException(ErrorCode errorCode, Object[] args, Throwable cause) {
        super(errorCode, args, cause);
    }
}

File: apps/auth-server/src/main/kotlin/com/example/chat/auth/server/common/exception/AuthException.kt

package com.example.chat.auth.server.common.exception

import com.example.chat.common.core.exception.BaseException
import com.example.chat.common.core.exception.ErrorCode

class AuthException(
    errorCode: ErrorCode,
    args: Array<out Any?>? =
        null,
    cause: Throwable? =
        null
) : BaseException(errorCode, args, cause)
